app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewItem.kt

```diff
@@ -0,0 +1,44 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock.getCrewman
+import agency.five.codebase.android.movieapp.model.Crewman
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+
+data class CrewItemViewState(
+    val name: String,
+    val job: String,
+        )
```

Comment:
This file is not properly formatted. You can format a file by using `Ctrl + Alt + L` on Windows, or `Command + Option + L` on Mac. Use this for all files.

Reply:
Fixed